lib/internal/per_context/primordials.js

```diff
@@ -541,11 +541,14 @@ primordials.SafePromiseAllReturnVoid = (promises, mapFn) =>
   new Promise((resolve, reject) => {
     let pendingPromises = promises.length;
     if (pendingPromises === 0) resolve();
+    const onPromiseSettled = () => {
```

Comment:
Let's use the same term as the spec
```suggestion
    const onFulfilled = () => {
```

Reply:
> Let's use the same term as the spec

Done 🤝

Reply:
> Let's use the same term as the spec

@aduh95 Done 🤝